service/service_test.go

```diff
@@ -38,7 +38,16 @@ import (
 func TestServiceCreation(t *testing.T) {
 	r := metrics.NewTestStatsReporter()
 	r.CountersWG.Add(1)
-	scope, closer := tally.NewRootScope("", nil, r, 50*time.Millisecond, tally.DefaultSeparator)
+	opts := tally.ScopeOptions{
+		Tags:           nil,
```

Comment:
why the explicit nil/empty string stuff here? i think i actually like it, but generally see it elided as:

```go
opts := tally.ScopeOptions {
  Reporter: r,
  Separator: ...,
  DefaultBuckets: ...,
}
```

for brevity. thoughts?